homework_3/barbhaiwala/solution.js

```diff
@@ -0,0 +1,83 @@
+function NaiveCalculator(equation)
+{
+    this.splitTokens = function()
```

Comment:
Unless you're constructing, no need to assign to `this`. You could rewrite this declaration as:
`function splitTokens () {`